### ISSUE TITLE
record webcam

### ISSUE THREAD

**Issue by geofflangenderfer** [SeaDve/Kooha#21]
Is it possible to record webcam and desktop together?

**Comment by SeaDve** [comment]
> Is it possible to record webcam and desktop together?

Nope, although I might consider this in the future. For now, I think OBS supports this feature.

**Comment by kinghat** [comment]
> Is it possible to record webcam and desktop together?

or record just the web cam 🤔 OBS would be for combining the recordings.

**Comment by maks** [comment]
a simple workaround for now could be to just show your webcam output in a window on the same desktop that you are recoding via Kooha.
eg. with `mpv` you could run `mpv av://v4l2:/dev/video0 --profile=low-latency --untimed` 
which would give you a decoration-less (once you remove mouse pointer over it) window of your webcam feed.

**Comment by SeaDve** [comment]
Planning this as part of the UI refresh; see https://github.com/SeaDve/Kooha/discussions/263
